subtests/docker_cli/load/load.py

```diff
@@ -26,7 +28,7 @@ class load(subtest.Subtest):
     def initialize(self):
         super(load, self).initialize()
         self.stuff['di'] = di = DockerImages(self)
-        img_id = self.config['test_id'].strip()
+        img_id = self._test_id
```

Comment:
oh it's a property, missed that.  Still don't quite follow, but I trust you, it's needed